I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    show screen button1
  File "renpy/common/000statements.rpy", line 510, in execute_show_screen
    renpy.show_screen(name, *args, **kwargs)
Exception: Screen button1 is not known.


-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 34, in script
    show screen button1
  File "C:\Users\Rodrigo\Downloads\renpy-7.2.1-sdk\renpy\ast.py", line 1907, in execute
    self.call("execute")
  File "C:\Users\Rodrigo\Downloads\renpy-7.2.1-sdk\renpy\ast.py", line 1895, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\Rodrigo\Downloads\renpy-7.2.1-sdk\renpy\statements.py", line 220, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 510, in execute_show_screen
    renpy.show_screen(name, *args, **kwargs)
  File "C:\Users\Rodrigo\Downloads\renpy-7.2.1-sdk\renpy\display\screen.py", line 1077, in show_screen
    raise Exception("Screen %s is not known.\n" % (name[0],))
Exception: Screen button1 is not known.


Windows-8-6.2.9200
Ren'Py 7.2.1.457
 1.0
Mon Mar 25 08:31:38 2019
